test/async/mandos/message_otherShard.scen.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "message_otherShard",
+    "steps": [
+        {
+            "step": "setState",
+            "accounts": {
+                "0xa94f5374fce5edbc8e2a8697c15331677e6ebffefefefefefefefefefefe1234": {
+                    "nonce": "0",
+                    "shard": "0",
+                    "balance": "0x10000000000000e8d4a51000",
+                    "storage": {},
+                    "code": ""
+                },
+                "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa1234": {
+                    "nonce": "0",
+                    "shard": "0",
+                    "balance": "0",
+                    "storage": {
+                        "0x0000000000000000000000000000000000000000000000000000000000000000": "0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb5555"
+                    },
+                    "code": "file:../async-alice/output/alice.wasm"
```

Comment:
I renamed these contracts and cleaned up the mandos tests a little bit, I can help you with the update.
They are now called proxy_test_first and proxy_test_second.
In any case, for testing arwen, I think at this point forwarder-raw + vault would be easier to use, since there is nothing hard-coded in these contracts.
Nothing has to be done immediately, just be aware that we have to sync them to the lastest version of the framework at some point.

Reply:
Good point. We should sync them after `feat/promises-final` is merged into `master.